hackathon_site/event/migrations/0006_profile_phone_number.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 3.2.15 on 2022-09-12 03:15
+
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("event", "0005_hardware_site_admins_v2"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="profile",
+            name="phone_number",
+            field=models.CharField(
+                default="8022818076",
+                max_length=20,
+                validators=[
```

Comment:
I think this migration is not up to date, could you remove the validators array (and the import) and set the default to an empty string so profiles don't accidentally have incorect phone numbers